get_rhcos_image.sh

```diff
@@ -6,3 +6,7 @@ source common.sh
 if [ ! -f "$RHCOS_IMAGE_FILENAME" ]; then
     curl --insecure --compressed -L -o "${RHCOS_IMAGE_FILENAME}" "${RHCOS_IMAGE_URL}/${RHCOS_IMAGE_VERSION}/${RHCOS_IMAGE_FILENAME}".gz
 fi
+
+if [ ! -f "${RHCOS_IMAGE_FILENAME_OPENSTACK}.gz" ]; then
+    curl --insecure --compressed -L -o "${RHCOS_IMAGE_FILENAME_OPENSTACK}.gz" "${RHCOS_IMAGE_URL}/${RHCOS_IMAGE_VERSION}/${RHCOS_IMAGE_FILENAME_OPENSTACK}.gz"
```

Comment:
This isn't related to the rest of the patch, and I'm not sure it's correct - doesn't --compressed decompress the gz on the fly so we end up with the uncompressed qcow?  Either way can we remove this and push another PR if necessary? Thanks! :)

Reply:
It is necessary - this is the image being mounted in the prebuilt container, as previously it was curl'ed during the build

Reply:
Then we either need to fix the filename in the container or remove the --compressed flag?

From the curl manpage:
```
--compressed
              (HTTP) Request a compressed response using one of the algorithms
              curl  supports,  and  save  the  uncompressed document
```

Reply:
Right, `--compressed` is useless, dropped it